src/bitmessagekivy/baseclass/login.py

```diff
@@ -71,11 +71,11 @@ def address_created_callback(self, dt=0):
     def reset_address_spinner(self):
         """reseting spinner address and UI"""
         addresses = [addr for addr in config.addresses()
-                     if config.get(str(addr), 'enabled') == 'true']
+                     if config.getboolean(str(addr), 'enabled')]
         self.manager.parent.ids.content_drawer.ids.identity_dropdown.values = []
-        self.manager.parent.ids.sc3.children[1].ids.identity_dropdown.values = []
+        # self.manager.parent.ids.sc3.children[1].ids.identity_dropdown.values = []
```

Comment:
these should be removed, we can add them later if needed.

Reply:
Done